packs/linux/actions/get_live_hosts.yaml

```diff
@@ -0,0 +1,11 @@
+name: get_live_hosts
+runner_type: run-python
+description: List all hosts on a given Network
+enabled: true
+entry_point: get_live_hosts.py
+parameters:
+  network:
+    type: string
+    description: "Network to query live hosts "
```

Comment:
Would be more explicit to say `Subnet of a network to scan for live hosts (e.g. 192.168.0.0/24)`. Probably also better to call the parameter `subnet`.